src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components'
+
+export type AvatarProps = {
+  name: string
+  size: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8
+}
+
+const Wrapper = styled.div<Omit<AvatarProps, 'name'>>`
+  ${({ size, theme }) => css`
+    width: calc(${size} * 1.6rem);
+    height: calc(${size} * 1.6rem);
+    background-color: ${theme.colors.darkBlack};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    font-size: calc((${size} * 1.6rem) / 2);
+    color: ${theme.colors.white};
+  `}
+`
+
+const Avatar = ({ name, size }: AvatarProps) => {
+  const arrOfLetters = name.split('')
+  const onlyLettersRegex = /^[A-Za-z]+$/
+  const firstLetter = arrOfLetters.find((letter) =>
```

Comment:
Essa regex tem um problema, ela considera somente chars sem acento. Se o nome for `Érica` a letra que ele vai pegar vai ser o R.